buildLogic/multiplatform-convention/src/main/kotlin/app/cash/sqldelight/multiplatform/MultiplatformConventions.kt

```diff
@@ -4,66 +4,59 @@ import org.gradle.api.Plugin
 import org.gradle.api.Project
 import org.jetbrains.kotlin.gradle.dsl.KotlinMultiplatformExtension
 import org.jetbrains.kotlin.gradle.dsl.kotlinExtension
-import org.jetbrains.kotlin.gradle.plugin.mpp.KotlinNativeTarget
-import org.jetbrains.kotlin.gradle.targets.js.KotlinJsTarget
 import org.jetbrains.kotlin.konan.target.HostManager
 
 class MultiplatformConventions : Plugin<Project> {
   override fun apply(project: Project) {
     project.plugins.apply("org.jetbrains.kotlin.multiplatform")
 
     (project.kotlinExtension as KotlinMultiplatformExtension).apply {
-      val nativeMain = sourceSets.create("nativeMain").apply {
-        dependsOn(sourceSets.getByName("commonMain"))
-      }
-      val nativeTest = sourceSets.create("nativeTest").apply {
-        dependsOn(sourceSets.getByName("commonTest"))
-      }
+      targetHierarchy.default()
 
-      targets.whenObjectAdded {
-        if (it is KotlinNativeTarget) {
-          sourceSets.getByName("${it.name}Main").dependsOn(nativeMain)
-          sourceSets.getByName("${it.name}Test").dependsOn(nativeTest)
-        }
+      jvm()
 
-        if (it is KotlinJsTarget) {
-          it.browser {
-            testTask {
-              useKarma {
-                useChromeHeadless()
-              }
+      js {
+        browser {
+          testTask {
+            useKarma {
+              useChromeHeadless()
             }
           }
-          it.compilations.all {
-            it.kotlinOptions.apply {
-              moduleKind = "umd"
-              sourceMap = true
-              sourceMapEmbedSources = null
-            }
+        }
+        compilations.configureEach {
+          it.kotlinOptions {
+            moduleKind = "umd"
+            sourceMap = true
+            sourceMapEmbedSources = null
           }
         }
```

Comment:
Pretty sure the top two are the default now

Reply:
Yes, they are.